docs/cn/advanced/Security.md

```diff
@@ -186,7 +186,7 @@ Alluxio审计日志格式如下表所示：
 
 它和HDFS审计日志的格式[wiki](https://wiki.apache.org/hadoop/HowToConfigure)很像。
 
-为了使用Alluxio的审计功能，你需要将JVM参数`alluxio.master.audit.logging.enabled`设置为true，具体可见[Configuration settings](Configuration-Settings.html)。
+为了使用Alluxio的审计功能，你需要将JVM参数`alluxio.master.audit.logging.enabled`设置为true，具体可见[Configuration settings]({{ '/cn/operation/Configuration.html' | relativize_url }})。
```

Comment:
Change to `/en/reference/Properties-List.html`?

Reply:
This one is also configuration.html